### DISCUSSION TITLE
Structured data for Google job search

### DISCUSSION THREAD

**Discussion by TimAtLinks** [bullhorn/career-portal#475] [Q&A] [answered]
Matador Jobs automatically optimizes job postings with structured data to directly integrate with Google, correct?

https://www.bullhorn.com/marketplace/matador-jobs/
https://jobs.google.com/about/
https://developers.google.com/search/docs/advanced/structured-data/job-posting

I'm trying to assist my IT Staffing team with the Google search performance (or lack thereof) of our job board:

https://careers.linkstechnology.com

Thanks,
Tim

**Comment by pbearne** [comment, marked as answer]
HI @TimAtLinks 
Yes, and provides options (replace company name with agency name)
Matador also uses the indexing API to push jobs to google in realtime  
https://developers.google.com/search/apis/indexing-api/v3/quickstart

You can reach for a demo via the marketplace

**Reply by TimAtLinks** [reply]
Perfect, thanks.

So, the structure is in place. Now, is there anything that we can do to show up more prevalently in Google Jobs? 🥺

(I know, probably fighting the usual fight against hundreds of factors and signals of varying weights that Google's search algorithm looks at to rank results ...)

**Reply by pbearne** [reply]
Make sure you provide the Salary range if you can see the docs how to add it https://matadorjobs.com/support/documentation/adding-salary-range-to-ldjson-data/
and set up the google indexing API settings

**Reply by TimAtLinks** [reply]
Whoa, that's what I was making sure of—that we didn't need to worry about structured data/JSON when using Bullhorn!

Why don't you provide a salary range object? I see that your documentation is a couple of years old. Is this feature on the roadmap?

I still don't know how/where to add salary range data after reading that documentation (maybe a developer will).

Also, I thought you already said that Matador uses Google's indexing API. 🤔 Or do you mean that API settings need to be updated after adding salary range ("set up" threw me off)?

**Reply by TimAtLinks** [reply]
Oh man, we're using https://bullhorn.github.io/career-portal/ ... Basic Static Host ... 😳

So, I guess Advanced hosting is a likely next step (or perhaps Matador Jobs, for real). I wonder if "Generate an XML sitemap" is good enough when Google explicitly recommends using their Indexing API:

https://developers.google.com/search/docs/advanced/structured-data/job-posting

**Reply by pbearne** [reply]
@TimAtLinks there is a reason sites install Matador Jobs all these little problems are sorted
The XML sitemap will list the jobs but not push the jobs in realtime so you are at the whim of when Google whats to re-index you
does speed matter to you how fast do your jobs update/get filled?

**Reply by TimAtLinks** [reply]
Fair enough. Can you use Matador Jobs outside of WordPress? Also, I can't easily find pricing, which I assume is more than free Career Portal. 😉

**Reply by pbearne** [reply]
The cost for Matador Jobs is https://matadorjobs.com/product/pricing/
Matador Jobs does need WordPress and that is one of it benefits as we get a load item for free from the WP instal
Ask for a demo for more info

Paul

**Reply by TimAtLinks** [reply]
We will check it out. Thanks for your timely responses.
